src/components/API.js

```diff
@@ -0,0 +1,152 @@
+export default class API {
+  constructor(options) {
+    this._options = options;
+    this._baseURL = options.baseUrl;
+    this._authtoken = options.headers.authorization;
+    this._contentTypeJSON = options.headers["Content-Type"];
+  }
+
+  getInitalCards() {
+    return fetch(`${this._baseURL}/cards`, {
+      method: "GET",
+      headers: {
+        authorization: this._authtoken,
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        return Promise.reject(res);
+      })
+      .catch((err) => {
+        console.error(`Error Loading Cards: ${err.status}`);
+      });
+  }
+
+  getUserInformation() {
+    return fetch(`${this._baseURL}/users/me`, {
+      method: "GET",
+      headers: {
+        authorization: this._authtoken,
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        return Promise.reject(res);
+      })
+      .catch((err) => {
+        console.error(`Error Loading Cards: ${err.status}`);
+      });
+  }
+
+  renderCardsValidator() {
+    return Promise.all([this.getInitalCards(), this.getUserInformation()]);
+  }
+
+  patchProfileInformation({ name, about }) {
+    return fetch(`${this._baseURL}/users/me`, {
+      method: "PATCH",
+      headers: {
+        authorization: this._authtoken,
+        "Content-Type": "application/json",
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)